doc/source/release-notes/Percona-Server-5.6.25-73.0.rst

```diff
@@ -0,0 +1,45 @@
+.. rn:: 5.6.25-73.0
+
+==============================
+ |Percona Server| 5.6.25-73.0 
+==============================
+
+Percona is glad to announce the release of |Percona Server| 5.6.25-73.0 on June 29th, 2015 (Downloads are available `here <http://www.percona.com/downloads/Percona-Server-5.6/Percona-Server-5.6.25-73.0/>`_ and from the :doc:`Percona Software Repositories </installation>`).
+
+Based on `MySQL 5.6.25 <http://dev.mysql.com/doc/relnotes/mysql/5.6/en/news-5-6-25.html>`_, including all the bug fixes in it, |Percona Server| 5.6.25-73.0 is the current GA release in the |Percona Server| 5.6 series. All of |Percona|'s software is open-source and free, all the details of the release can be found in the `5.6.25-73.0 milestone at Launchpad <https://launchpad.net/percona-server/+milestone/5.6.24-72.2>`_.
+
+New Features
+============
+
+ |Percona Server| has implemented :ref:`proxy_protocol_support`. The implementation is based on a patch developed by Thierry Fournier.
+ 
+Bugs Fixed
+==========
+
+ Symlinks to ``libmysqlclient`` libraries were missing on *CentOS* 6. Bug fixed :bug:`1408500`.
+
+ *RHEL/CentOS* 6.6 ``openssl`` package (``1.0.1e-30.el6_6.9``), containing a fix for CVE-2015-4000, changed the DH key sizes to a minimum of 768 bits. This caused an issue for *MySQL* as it uses 512 bit keys. Fixed by backporting an upstream 5.7 fix that increases the key size to 2048 bits. Bug fixed :bug:`1462856` (upstream :mysqlbug:`77275`).
+ 
+ InnoDB's checksum algorithm would incorrectly declare a page corrupted in case the page was not empty and the lsn was ``0``, which would cause an assertion. Bug fixed :bug:`1467760` (upstream :mysqlbug:`73689`) *Justin Tolmer*.
```

Comment:
Some compressed InnoDB data pages could be mistakenly considered corrupted, crashing the server. Bug fixed ...